recipes/stormevents/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "stormevents" %}
+{% set version = "1.3.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stormevents-{{ version }}.tar.gz
+  sha256: 0b7ea88e22ae1d72a2be3b49f00045eeb85671c5e92783a1bdbf72db76891bd5
+
+build:
+  noarch: python
+  script_env:
+    - __version__={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - dunamai
+    - pip
+    - python >=3.6
+    - setuptools >=41.2
+  run:
+    - beautifulsoup4
+    - geopandas
+    - netcdf4
+    - numpy
+    - pandas
+    - pyproj >=2.6
+    - python >=3.6
+    - python-dateutil
+    - requests
+    - shapely
+    - typepigeon >=1.0.5
+    - xarray
+
+test:
+  imports:
+    - stormevents
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/noaa-ocs-modeling/StormEvents.git
```

Should this actually be:
```suggestion
  home: https://github.com/oceanmodeling/StormEvents
```
The original URL gives me a 404. It would also do this for a private repo. The suggested URL corresponds to what the PyPI project links to, for docs at least.

---

thank you, I did not catch that